experimental/packages/opentelemetry-sdk-metrics-base/src/MeterProvider.ts

```diff
@@ -46,11 +46,9 @@ export class MeterProvider {
         return metrics.NOOP_METER;
     }
 
-    // Spec leaves it unspecified if creating a meter with duplicate
-    // name/version returns the same meter. We create a new one here
-    // for simplicity. This may change in the future.
-    // TODO: consider returning the same meter if the same name/version is used
-    return new Meter(this._sharedState, { name, version, schemaUrl: options.schemaUrl });
+    const meter = new Meter(this._sharedState, { name, version, schemaUrl: options.schemaUrl });
+    this._sharedState.meters.push(meter);
```

Comment:
Should this be in the `Meter` constructor?

Reply:
updated :)